playbooks/manageiq-providers-openstack-test-public-clouds/run.yaml

```diff
@@ -0,0 +1,77 @@
+- hosts: all
+  become: yes
+  roles:
+    - export-cloud-openrc
+  tasks:
+    - name: Run Integration tests of Docker machine against devstack
```

Comment:
please modify the task name, it's tests of manageiq not Docker
machine.

Reply:
done, thx